### PR TITLE
Update sandman_de.ts

### PR DESCRIPTION
**Pull request by bastik-1001**
Fresh fork to translate the new strings.

@APMichael, if you can please have a look.

There are some mistakes in the source, so I guess I might have to touch the file again. (Like the spelling of sellected, or sellect, where there is an additional l.)

### CODE REVIEW

File: SandboxiePlus/SandMan/sandman_de.ts [hunk 1]
```diff
@@ -6,121 +6,121 @@
...
-        <translation type="unfinished">Eine Sandbox isoliert Ihr Hostsystem von Prozessen, die in dieser Box laufen, sie hindert diese permanente Änderungen an anderen Programmen oder Daten auf Ihrem Computer zu machen. Der Level an Isolation beeinflusst Ihre Sicherheit, ebenso wie die Kompatibilität mit Programmen, somit wird es verschiedene Stufen der Isolation geben, abhängig vom ausgewählten Boxtyp. Sandboxie kann auch Ihre persönlichen Daten vor Zugriff durch Prozesse, die unter dessen Aufsicht laufen, schützen.</translation>
+        <translation>Eine Sandbox isoliert Ihr Hostsystem von Prozessen, die in dieser Box laufen, sie hindert diese permanente Änderungen an anderen Programmen oder Daten auf Ihrem Computer zu machen. Der Level an Isolation beeinflusst Ihre Sicherheit, ebenso wie die Kompatibilität mit Programmen, somit wird es verschiedene Stufen der Isolation geben, abhängig vom ausgewählten Boxtyp. Sandboxie kann auch Ihre persönlichen Daten vor Zugriff durch Prozesse, die unter dessen Aufsicht laufen, schützen.</translation>
```
Comment (APMichael) [inline SandboxiePlus/SandMan/sandman_de.ts:73]: I would write "..., sie hindert diese **daran** permanente Änderungen...".

File: SandboxiePlus/SandMan/sandman_de.ts [hunk 2]
```diff
@@ -220,73 +220,74 @@
...
         <location filename="Wizards/NewBoxWizard.cpp" line="361"/>
         <source>The sellected box location is not a valid path.</source>
-        <translation type="unfinished"></translation>
+        <translation>Der ausgewählte Boxspeicherort ist kein gültiger Pfad.</translation>
     </message>
     <message>
         <location filename="Wizards/NewBoxWizard.cpp" line="366"/>
         <source>The sellected box location exists and is not empty, it is recomended to pick a new or empty folder. Are you sure you want to use an existing folder?</source>
-        <translation type="unfinished"></translation>
+        <translation>Der ausgewählte Boxspeicherort existiert und ist nicht leer, es wird empfohlen einen neuen oder leeren Ordner auszuwählen. Sind Sie sicher, dass Sie einen exitierenden Ordner verwenden möchten?</translation>
```
Comment (APMichael) [inline SandboxiePlus/SandMan/sandman_de.ts:285]: Typo: "..., dass Sie einen **existierenden** Ordner verwenden..."

File: SandboxiePlus/SandMan/sandman_de.ts [hunk 3]
```diff
@@ -220,73 +220,74 @@
...
     </message>
     <message>
         <location filename="Wizards/NewBoxWizard.cpp" line="371"/>
         <source>The sellected box location not placed on a currently available drive.</source>
-        <translation type="unfinished"></translation>
+        <translation>Der ausgewählte Boxspeicherort befindet sich auf einen momentan nicht verfügbaren Laufwerk.</translation>
```
Comment (APMichael) [inline SandboxiePlus/SandMan/sandman_de.ts:290]: Typo: "...befindet sich auf **einem** momentan nicht..."

**Comment by APMichael** [comment]
> @APMichael, if you can please have a look.

Looks good to me. 👍 I have commented on the 3 minor typos.

File: SandboxiePlus/SandMan/sandman_de.ts (hunk 1 above)
Comment (bastik-1001) [reply SandboxiePlus/SandMan/sandman_de.ts]: Sounds complete now. Thank you.

File: SandboxiePlus/SandMan/sandman_de.ts (hunk 3 above)
Comment (bastik-1001) [reply SandboxiePlus/SandMan/sandman_de.ts]: Grammar mistake has been fixed.

**Comment by bastik-1001** [comment]
I fixed those things, including the typo you found in the previous pull request.

**Pull request merged**